working-groups/WORKING-GROUPS.md

```diff
@@ -319,6 +319,15 @@ Security concerns across Knative components
 
 
 
+## Event Delivery
+
+| &nbsp;                                                          | Leads               | Profile                                               | Duration  |
+| --------------------------------------------------------------- | ------------------- | ----------------------------------------------------- | --------- |
+| <img width="30px" src="https://github.com/matzew.png">          | Matthias Wessendorf | [matzew](https://github.com/matzew)                   | 2019-2021 |
```

Comment:
Since this was originally Eventing Channels WG that then became Event Delivery (or am I misremembering?), you should add @Harwayne here as well, and maybe even add a blurp that it was known as Eventing Channels?
https://github.com/knative/community/commit/8ca15d2ff226a5ce3b1b7ea6a7b4be8609e6b4af#diff-d2dbeea6c878da197e644ffa2772c52905c2313a18ae4b5b6a4bfcf96e233eeb

Reply:
Correct, precisely it was called "Channels Working Group" https://docs.google.com/document/d/1cSTc9FA9DWS198OOQeUUHOicnvM0jAQNPMb_Ln297jc/edit#